fastapi/fake-llm-openai/fake_llm_server.py

```
@@ -187,6 +187,24 @@ async def embeddings(request: Request):
     return JSONResponse(response)
 
 
+@app.get("/v1/models")
+async def list_models():
+    # OpenBayes 的 nanny 依次探测 /models 和 /v1/models 来判断这是不是一个 LLM
+    # serving，探到了才会装 litellm 并挂上 Langfuse callback。少了这个端点，
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)